ivy_tests/test_ivy/test_frontends/test_jax/test_nn/test_non_linear_activations.py

```diff
@@ -191,7 +189,7 @@ def test_jax_gelu(
 ):
     input_dtype, x = dtype_and_x
     # As erf function doesn't support complex dtype
-    if "complex" in str(x[0].dtype):
+    if "complex" in str(input_dtype):
```

Comment:
worth confirming, but probably `input_dtype` is a `list[str]` rather than a `str`

Reply:
Yeah, you're right. `str(input_dtype)` does still work since it returns `"['complex128']"` (for example) but I guess it's more clear to do `input_dtype[0]` directly

Reply:
sounds good, thanks!